fastlane_core/lib/fastlane_core/tool_collector.rb

```diff
@@ -0,0 +1,72 @@
+module FastlaneCore
+  class ToolCollector
+    TOOLS = %w(fastlane fastlane_core deliver snapshot frameit pem sigh produce cert gym pilot credentials_manager spaceship scan supply watchbuild match screengrab)
```

Comment:
I'd prefer if `fastlane_core` doesn't know about the tools. Looking at the code here, it seems like we don't need to actually check for `is_official?`. 

Is there any situation where we call `did_launch_action` in `tool_collector` and the `name` is an unofficial action? I guess only when we decide to use it in `fastlane` too (comment https://github.com/fastlane/fastlane/pull/4473/files#r61852911)

If we do, we could do the check in `fastlane`, instead of here.
What do you think?